common/src/main/scala/gu/msnotifications/package.scala

```diff
@@ -1,24 +1,24 @@
 package gu
 
-import java.util.Base64
-
-import models.{TopicType, Topic, UserId}
+import models.{Topic, UserId}
 
 package object msnotifications {
 
-  private def encode(string: String): String =
-    Base64.getUrlEncoder.encodeToString(string.getBytes("UTF-8"))
+  private def encodeBase16(string: String): String =
```

Comment:
Is there a better variable that could be used instead of "string"?  To me, even "s" would be an improvement